Formula/systemd.rb

```diff
@@ -0,0 +1,55 @@
+class Systemd < Formula
+  desc "System and service manager"
+  homepage "https://wiki.freedesktop.org/www/Software/systemd/"
+  url "https://github.com/systemd/systemd/archive/v246.tar.gz"
+  sha256 "4268bd88037806c61c5cd1c78d869f7f20bf7e7368c63916d47b5d1c3411bd6f"
+  license all_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
+  head "https://github.com/systemd/systemd.git"
+
+  depends_on "coreutils" => :build
+  depends_on "docbook-xsl" => :build
+  depends_on "gettext" => :build
+  depends_on "gperf" => :build
+  depends_on "intltool" => :build
+  depends_on "libgpg-error" => :build
+  depends_on "libtool" => :build
+  depends_on "libxslt" => :build
+  depends_on "m4" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+  depends_on "expat"
+  depends_on "libcap"
+  depends_on :linux
+  depends_on "lz4"
+  depends_on "openssl@1.1"
+  depends_on "util-linux" # for libmount
+  depends_on "xz"
+
+  def install
+    args = %W[
+      --prefix=#{prefix}
+      --libdir=lib
```

Comment:
Is this correct?

Reply:
If you're asking about `--libdir=lib`, I think I added this to prevent `systemd` from installing to `lib64` on RedHat systems.  That argument is just the folder name, not the full path.